src/main/webapp/WEB-INF/tags/instructor/results/filterEditModal.tag

```diff
@@ -0,0 +1,119 @@
+<%@ tag description="instructorFeedbackResultsTop - Filter Panel Edit Modal" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ attribute name="filterPanel" type="teammates.ui.template.InstructorFeedbackResultsFilterPanel" required="true" %>
+
+<div id="editModal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title">Edit settings</h4>
+      </div>
+      <div class="modal-body">
+        <form role="form" method="post" action="${filterPanel.resultsLink}">
```

Comment:
in HTML5, the ARIA attribute `role="form"` is already [implicitly set](https://www.w3.org/TR/html-aria/#dfn-implicit-aria-semantics), so it doesn't have to be mentioned again.